src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -299,13 +299,47 @@ public void SendFunctionLoadRequests(ManagedDependencyOptions managedDependencyO
                     _eventSubscriptions.Add(_inboundWorkerEvents.Where(msg => msg.MessageType == MsgType.FunctionLoadResponse)
                         .Subscribe((msg) => LoadResponse(msg.Message.FunctionLoadResponse), HandleWorkerFunctionLoadError));
                 }
-                foreach (FunctionMetadata metadata in _functions.OrderBy(metadata => metadata.IsDisabled()))
+
+                // Check if the worker supports this feature
+                bool capabilityEnabled = !string.IsNullOrEmpty(_workerCapabilities.GetCapabilityState(RpcWorkerConstants.AcceptsListOfFunctionLoadRequests));
+                if (capabilityEnabled)
+                {
+                    SendFunctionLoadRequestsList(_functions, managedDependencyOptions);
```

Comment:
naming nit: function name sounds like it sends multiple requests. but in reality, it sends only one grpc request. Should we use `SendFunctionLoadRequest`?

Reply:
I think `SendFunctionLoadRequestCollection / List` is good (removing s from requests